README.md

```diff
@@ -42,6 +42,7 @@ To learn more about the docker pull rate limits and the open source software pro
 | [bcftools](https://hub.docker.com/r/staphb/bcftools/) <br/> [![docker pulls](https://badgen.net/docker/pulls/staphb/bcftools)](https://hub.docker.com/r/staphb/bcftools) | <ul><li>1.10.2</li><li>1.11</li><li>1.12</li><li>1.13</li><li>1.14</li><li>1.15</li></ul> | https://github.com/samtools/bcftools |
 | [bedtools](https://hub.docker.com/r/staphb/bedtools/) <br/> [![docker pulls](https://badgen.net/docker/pulls/staphb/bedtools)](https://hub.docker.com/r/staphb/bedtools) | <ul><li>2.29.2</li><li>2.30.0</li></ul> | https://bedtools.readthedocs.io/en/latest/ <br/>https://github.com/arq5x/bedtools2 |
 | [berrywood-report-env](https://hub.docker.com/r/staphb/berrywood-report-env/) <br/> [![docker pulls](https://badgen.net/docker/pulls/staphb/berrywood-report-env)](https://hub.docker.com/r/staphb/berrywood-report-env) | <ul><li>1.0</li></ul> | none |
+| [blast+](https://hub.docker.com/r/staphb/blast/) <br/> [![docker pulls](https://badgen.net/docker/pulls/staphb/blast)](https://hub.docker.com/r/staphb/blast) | <ul><li>2.13.0</li></ul> | https://www.ncbi.nlm.nih.gov/books/NBK279690/ <br/> https://www.ncbi.nlm.nih.gov/books/NBK279690/ |
```

Comment:
Was it an accident to copy the ncbi URL twice here? Did you mean to paste a different URL?

https://www.ncbi.nlm.nih.gov/books/NBK279690/

Reply:
Sorry!